metricbeat/module/mongodb/_meta/config.yml

```diff
@@ -1,9 +1,9 @@
-- module: mongodb
-  metricsets: ["status"]
-  enabled: true
-  period: 10s
+#- module: mongodb
+  #metricsets: ["status"]
+  #enabled: true
```

Comment:
If we have `enabled: false` available here should we just use that instead of commenting out? // cc: @ruflin